source/index.html.md

```diff
@@ -825,6 +860,47 @@ status_eq | Show jobs in a certain status
 status_in | Show jobs in a group of statuses
 status_not_eq | Show jobs that are not in a certain status
 
+## View a Single Job
```

Comment:
how are we dealing with leads? Are we going to add that later?

Reply:
yeah when it's fully built and we want to expose it publicly we'll add it in.